src/Tool/Lerna.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Silverorange\PackageRelease\Tool;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Silverorange\PackageRelease\Console\ProcessRunner;
+
+/**
+ * @package   PackageRelease
+ * @author    Meg Mitchell <meg@silverorange.com>
+ * @copyright 2020 silverorange
+ * @license   http://www.opensource.org/licenses/mit-license.html MIT License
+ */
+class Lerna
+{
+    public static function verify(OutputInterface $output): bool
+    {
+        $command = 'yarn global add lerna';
```

Comment:
we shouldn't need this anymore. The lerna package is now installed system-wide on dev servers